src/utils/line.ts

```diff
@@ -788,7 +790,7 @@ export const formatLineCategories = (lines: { [key: string]: SecuredLine }): { [
   );
 
   const newCreditLines = _.map(
-    _.sortBy(lines, (line) => line.start),
+    _.sortBy(lines, (line) => -line.start),
```

Comment:
should already be sorted in the subgraph query/reponse

Reply:
this is so that if a user proposes a new line it is the first one we see on the Market page

Reply:
commented this out